specs/deneb/beacon-chain.md

```diff
@@ -279,16 +253,17 @@ def verify_and_notify_new_payload(self: ExecutionEngine,
     execution_payload = new_payload_request.execution_payload
     parent_beacon_block_root = new_payload_request.parent_beacon_block_root
 
-    # [Modified in Deneb:EIP4788]
-    if not self.is_valid_block_hash(execution_payload, parent_beacon_block_root):
+    # [New in Deneb:EIP4844]
+    assert parent_beacon_block_root == execution_payload.parent_beacon_block_root
```

Comment:
This should be in `process_execution_payload` with the other CL<>EL consistency checks